README.md

```diff
@@ -26,15 +30,27 @@ See [GitHunt-API](https://github.com/apollostack/GitHunt-API) and [GitHunt-React
 - `id: string` : the subscription ID of the subscription to unsubscribe from
 
 ## Server
-### `Constructor(options, httpServer)`
+### `Constructor(options, socketOptions)`
 - `options: {ServerOptions}`
   * `subscriptionManager: SubscriptionManager` : GraphQL subscription manager
-  * `onSubscribe?: (message: SubscribeMessage, params: SubscriptionOptions, webSocketRequest: WebSocketRequest)` : optional method to create custom params that will be used when resolving this subscription
+  * `onSubscribe?: (message: SubscribeMessage, params: SubscriptionOptions, webSocket: WebSocket)` : optional method to create custom params that will be used when resolving this subscription
+  * `onUnsubscribe?: (webSocket: WebSocket)` : optional method that called when a client unsubscribe
+  * `onConnect?: (connectionParams: Object, webSocket: WebSocket)` : optional method that called when a client connects to the socket, called with the `connectionParams` from the client, the return value of this callback will be available as part of the `context` of the subscription. return `false` or throw and exception to reject the connection.
```

Comment:
and be more specific about how the result is used: "if the return value is an object, its elements will be added to the `context`" or something